### PR TITLE
Added configuration examples for new merge policies

### PR DESCRIPTION
**Pull request by mmedenjak**
Also updated license headers to 2018.

Will backport to 3.10 and 3.10.x once it's approved and merged.

### CODE REVIEW

File: helper/src/main/java/com/hazelcast/examples/helper/CommonUtils.java [hunk 1]
```diff
@@ -219,7 +235,7 @@ public static void closeQuietly(Closeable closeable) {
         try {
             closeable.close();
         } catch (IOException ignored) {
-            EmptyStatement.ignore(ignored);
+            ignore(ignored);
```
Comment (kwart) [inline helper/src/main/java/com/hazelcast/examples/helper/CommonUtils.java]: Nitpicking: This doesn't belong to "Update license header" commit

File: hazelcast-integration/amazon-ec2-vagrant-chef/README.md [hunk 2]
```diff
@@ -212,7 +212,7 @@ INFO: [192.168.0.19]:5701 [dev] [3.4-RC1-SNAPSHOT] Starting with 2 generic opera
 Dec 31, 2014 2:53:22 PM com.hazelcast.system
 INFO: [192.168.0.19]:5701 [dev] [3.4-RC1-SNAPSHOT] Hazelcast 3.4-RC1-SNAPSHOT (20141215 - 24f7902) starting at A$
 Dec 31, 2014 2:53:22 PM com.hazelcast.system
-INFO: [192.168.0.19]:5701 [dev] [3.4-RC1-SNAPSHOT] Copyright (C) 2008-2014 Hazelcast.com
+INFO: [192.168.0.19]:5701 [dev] [3.4-RC1-SNAPSHOT] Copyright (C) 2008-2018 Hazelcast.com
```
[the remark this answers is not in the document]
Comment (mmedenjak) [reply hazelcast-integration/amazon-ec2-vagrant-chef/README.md]: I just search-replaced everything, nice to see it was very thorough :)

**Comment by mmedenjak** [comment]
Thanks Josef, David actually made a PR with code samples where we split and heal the cluster - https://github.com/hazelcast/hazelcast-code-samples/pull/221.
But some of us found it too complex for users to follow so we decided not to do it.
Maybe we can add it again some time or have a separate repo where a user can try out some complex scenario or play with different use cases.

**Comment by mmedenjak** [comment]
run-lab-run

**Pull request merged**